new_project/examples/separate_apps/src/lib.ts

```diff
@@ -0,0 +1,50 @@
+import { BitcoinWallet, Cnd, ComitClient, EthereumWallet } from "comit-sdk";
+import fs from "fs";
+
+export async function startClient(index: number): Promise<Actor> {
+    const bitcoinWallet = await BitcoinWallet.newInstance(
+        "regtest",
+        process.env.BITCOIN_P2P_URI!,
+        process.env[`BITCOIN_HD_KEY_${index}`]!
+    );
+    await new Promise(r => setTimeout(r, 1000));
```

Comment:
:question: What was the reason for this sleep again?

Reply:
To print the correct Bitcoin balance